### PR TITLE
Fix VS Code task for running Databroker CLI

### PR DESCRIPTION
**Pull request by BjoernAtBosch**

### CODE REVIEW

**Review by dennismeister93 — DISMISSED** [review]
lgtm

**Review by dennismeister93 — DISMISSED** [review]
lgtm

File: manifest.json [hunk 1]
```diff
@@ -7,7 +7,10 @@
             "programs": [
                 {
                     "id": "run-vehicledatabroker-cli",
-                    "executable": "./runtime_local/src/run-vehicledatabroker-cli.sh"
+                    "executable": "python3",
+                    "args": [
+                        "./runtime_local/src/run-vehicledatabroker-cli.py"
+                    ]
```
Comment (dennismeister93) [inline manifest.json]: if https://github.com/eclipse-velocitas/cli/pull/291 is fine you could add "interactive": true

Comment (BjoernAtBosch) [reply manifest.json]: Done

**Review by dennismeister93 — APPROVED** [review]
lgtm^3

**Pull request merged**